$NetBSD: patch-ab,v 1.4 2001/04/11 21:09:34 hubertf Exp $

From nobu@tech.isac.co.jp Wed Apr 11 23:06:35 2001
Date: Tue, 10 Apr 2001 20:34:57 +0900
From: Nobuhiro Yasutomi <nobu@tech.isac.co.jp>
To: hubertf@netbsd.org
Cc: nobu@psrc.isac.co.jp, pkg-manager@netbsd.org
Subject: Re: pkg/12596

Hello, Hubert.

Thank you for quick following.

There is found last minute bug in this package. Please add
following patch. (It's announce by source code auther)

This fix was only this time. Next update will be removed.
--
Nobuhiro Yasutomi. ISAC, Inc.

pkgsrc/misc/fd/patches/patch.ab:
diff -urNP ../FD-1.03r/browse.c ./browse.c
--- ../FD-1.03r/browse.c	Tue Apr  3 00:00:00 2001
+++ ./browse.c	Tue Apr 10 00:00:00 2001
@@ -1305,6 +1305,7 @@
 				}
 				else if (_chdir2(cp) >= 0) strcpy(tmp, cp);
 				else {
+					*fullpath = '\0';
 					warning(-1, cp);
 					def = NULL;
 					strcpy(fullpath, origpath);
